src/api/pokeAPI.ts

```diff
@@ -0,0 +1,37 @@
+type ResultsElement = {
+  name: string;
+  url: string;
+};
+
+export async function pokeAPI(searchQuery: string) {
+  let url = '';
+
+  if (searchQuery === '') {
+    url = 'https://pokeapi.co/api/v2/pokemon?limit=5&offset=0';
+  } else {
+    const encodeSearchQuery = encodeURIComponent(searchQuery.toLowerCase());
+    url = `https://pokeapi.co/api/v2/pokemon/${encodeSearchQuery}`;
+  }
```

Comment:
Lets separate logic for the request with search query string and without it.
Leave here only this line: 
`const url: string = 'https://pokeapi.co/api/v2/pokemon/${encodeSearchQuery}'`